vignettes/get_started.Rmd

```diff
@@ -1,20 +1,18 @@
 ---
-title: "Using cchsflow"
+title: "Get started"
 output: rmarkdown::html_vignette
+date: "`r Sys.Date()`"
 vignette: >
   %\VignetteIndexEntry{1 - using cchsflow}     
   %\VignetteEngine{knitr::rmarkdown} 
   %\VignetteEncoding{UTF-8}
 ---
 
 ```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(echo = TRUE, collapse = TRUE, comment = "#>")
 ```
 
-## Getting Started
-
-### Loading the necessary packages
-
+## Load packages
```

Comment:
Should we also include load datasets in this section as well?
I was thinking adding pseudocode to instruct users to load the CCHS datasets that are saved on their computer into the environment. Something like this:
`cchs2001 <- read.csv("~/Documents/cchs2001.csv")`

Reply:
Good idea.

Reply:
I'll add that.